Git-commit: 867abb88b91f548dfdc876b5873226421af722c4
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 30 Dec 2020 20:00:38 +0000
Subject: [PATCH] overlays: Rebuild "upstream" with latest ovmerge
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

The latest ovmerge drops disabled fragments, causing the "upstream"
overlay to change.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 .../boot/dts/overlays/upstream-overlay.dts    | 38 ++++-------
 .../dts/overlays/upstream-pi4-overlay.dts     | 66 +++++--------------
 2 files changed, 31 insertions(+), 73 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/upstream-overlay.dts b/arch/arm/boot/dts/overlays/upstream-overlay.dts
index 2e9dcd4f5f0a..7c4071a7cb27 100644
--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -8,96 +8,84 @@
 / {
 	compatible = "brcm,bcm2835";
 	fragment@0 {
-		target = <&cma>;
-		__dormant__ {
-			size = <0x10000000>;
-		};
-	};
-	fragment@1 {
 		target = <&i2c2>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@2 {
+	fragment@1 {
 		target = <&fb>;
 		__overlay__ {
 			status = "disabled";
 		};
 	};
-	fragment@3 {
+	fragment@2 {
 		target = <&pixelvalve0>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@4 {
+	fragment@3 {
 		target = <&pixelvalve1>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@5 {
+	fragment@4 {
 		target = <&pixelvalve2>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@6 {
+	fragment@5 {
 		target = <&hvs>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@7 {
+	fragment@6 {
 		target = <&hdmi>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@8 {
+	fragment@7 {
 		target = <&v3d>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@9 {
+	fragment@8 {
 		target = <&vc4>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@10 {
+	fragment@9 {
 		target = <&clocks>;
 		__overlay__ {
 			claim-clocks = <BCM2835_PLLD_DSI0 BCM2835_PLLD_DSI1 BCM2835_PLLH_AUX BCM2835_PLLH_PIX>;
 		};
 	};
-	fragment@11 {
+	fragment@10 {
 		target = <&vec>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@12 {
+	fragment@11 {
 		target = <&txp>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@13 {
-		target = <&hdmi>;
-		__dormant__ {
-			dmas;
-		};
-	};
-	fragment@14 {
+	fragment@12 {
 		target = <&audio>;
 		__overlay__ {
 			brcm,disable-hdmi;
 		};
 	};
-	fragment@15 {
+	fragment@13 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
diff --git a/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts b/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts
index 6195e02bf9ff..e0f7cc8ba9ea 100644
--- a/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts
@@ -8,144 +8,114 @@
 / {
 	compatible = "brcm,bcm2835";
 	fragment@0 {
-		target = <&cma>;
-		__dormant__ {
-			size = <0x10000000>;
-		};
-	};
-	fragment@1 {
 		target = <&ddc0>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@2 {
+	fragment@1 {
 		target = <&ddc1>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@3 {
+	fragment@2 {
 		target = <&hdmi0>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@4 {
+	fragment@3 {
 		target = <&hdmi1>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@5 {
+	fragment@4 {
 		target = <&hvs>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@6 {
+	fragment@5 {
 		target = <&pixelvalve0>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@7 {
+	fragment@6 {
 		target = <&pixelvalve1>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@8 {
+	fragment@7 {
 		target = <&pixelvalve2>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@9 {
+	fragment@8 {
 		target = <&pixelvalve3>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@10 {
+	fragment@9 {
 		target = <&pixelvalve4>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@11 {
+	fragment@10 {
 		target = <&v3d>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@12 {
+	fragment@11 {
 		target = <&vc4>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@13 {
+	fragment@12 {
 		target = <&txp>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@14 {
+	fragment@13 {
 		target = <&fb>;
 		__overlay__ {
 			status = "disabled";
 		};
 	};
-	fragment@15 {
+	fragment@14 {
 		target = <&firmwarekms>;
 		__overlay__ {
 			status = "disabled";
 		};
 	};
-	fragment@16 {
+	fragment@15 {
 		target = <&vec>;
 		__overlay__ {
 			status = "disabled";
 		};
 	};
-	fragment@17 {
-		target = <&hdmi0>;
-		__dormant__ {
-			dmas;
-		};
-	};
-	fragment@18 {
-		target = <&hdmi1>;
-		__dormant__ {
-			dmas;
-		};
-	};
-	fragment@19 {
+	fragment@16 {
 		target = <&audio>;
 		__overlay__ {
 			brcm,disable-hdmi;
 		};
 	};
-	fragment@20 {
+	fragment@17 {
 		target = <&dvp>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@21 {
-		target = <&pixelvalve3>;
-		__dormant__ {
-			status = "okay";
-		};
-	};
-	fragment@22 {
-		target = <&vec>;
-		__dormant__ {
-			status = "okay";
-		};
-	};
-	fragment@23 {
+	fragment@18 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-- 
2.31.1

